src/main/java/org/jabref/gui/commonfxcontrols/FieldFormatterCleanupsPanel.java

```diff
@@ -106,7 +106,13 @@ private void setupCombos() {
     }
 
     private void setupBindings() {
-        cleanupsEnabled.selectedProperty().bindBidirectional(cleanupsDisableProperty());
+        // cannot bindBidirectional, because viewModel.cleanupsDisableProperty().not() provides a read only
+        // BooleanBinding, but bindBidirectional requires a simple BooleanProperty
+        cleanupsEnabled.selectedProperty().addListener((observableValue, oldValue, newValue) ->
```

Comment:
Please replace this custom binding with https://github.com/JabRef/jabref/blob/master/src/main/java/org/jabref/gui/util/BindingsHelper.java#L62

The manual listeners may result in a race condition, where the update of cleanupsDisableProperty results in an update of cleanupsEnabled which then again updates cleanupsDisableProperty  etc. To prevent this, a locked update mechanism is used in the above `bindBidirection`: https://github.com/JabRef/jabref/blob/master/src/main/java/org/jabref/gui/util/BindingsHelper.java#L227